JS/test/airport.spec.js

```diff
@@ -66,30 +66,28 @@ describe('My Test', () => {
             }
```

Comment:
Please correct this code. There is an additional logic implemented. It can be written shorter (almost in one line).

Reply:
My test was removed